.github/workflows/build_image.yml

```diff
@@ -0,0 +1,44 @@
+name: Build Docker Image
+
+on:
+  push:
+    branches:
+      - next
+      - master
+      
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    name: "Deploy image on Dockerhub"
+    steps:
+      - name: Check out
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
```

Comment:
Github checkout retrieve last commit only, By specifying `0` you will pull the whole branch including tags.
Since we do not need it, we should remove this line also.
```suggestion
```